src/binaryutils.cpp

```diff
@@ -461,6 +461,35 @@ static Status convertNativeFileFormatRequestTensorToOVTensor(const TensorType& s
     return StatusCode::OK;
 }
 
+Status convertStringTensorToOVTensor(
+    const KFSTensorInputProto& src,
+    ov::Tensor& tensor,
+    const std::shared_ptr<TensorInfo>& tensorInfo,
+    const std::string* buffer) {
+    if (nullptr != buffer) {
+        SPDLOG_ERROR("STRING input should be located in bytes_contents field.");
```

Comment:
```suggestion
        SPDLOG_DEBUG("STRING input should be located in bytes_contents field.");
```
We should not bloat console with user level errors